card/lib/card/name.rb

```diff
@@ -5,52 +5,55 @@
 
 class Card
   class Name < SmartName
-
     self.params  = Card::Env # yuck!
     self.session = proc { Card::Auth.current.name }
     self.banned_array = ['/']
 
     def star?
-      simple? and '*' == s[0,1]
+      simple? && '*' == s[0, 1]
     end
 
     def rstar?
-      right and '*' == right[0,1]
+      right && '*' == right[0, 1]
     end
 
     def trait_name? *traitlist
       junction? && begin
         right_key = right_name.key
         !!traitlist.find do |codename|
-          card_id = Card::Codename[ codename ] and card = Card.fetch( card_id, skip_modules: true, skip_virtual: true ) and
+          card_id = Card::Codename[ codename ] and card = Card.fetch(card_id, skip_modules: true, skip_virtual: true) and
             card.key == right_key
         end
       end
     end
 
     def trait_name tag_code
-      card_id = Card::Codename[ tag_code ] and card = Card.fetch( card_id, skip_modules: true, skip_virtual: true ) and
+      card_id = Card::Codename[ tag_code ] and card = Card.fetch(card_id, skip_modules: true, skip_virtual: true) and
```

Comment:
Line is too long. [117/80]<br>Use `&&` instead of `and`.<br>Space inside square brackets detected.